arcgis-ios-sdk-samples/Content Display Logic/Controllers/OptionsTableViewController.swift

```diff
@@ -50,7 +50,7 @@ class OptionsTableViewController: UITableViewController {
     
     override func viewDidLoad() {
         super.viewDidLoad()
-        tableView.register(OptionCell.self, forCellReuseIdentifier: "OptionCell")
+        tableView.register(UITableViewCell.self, forCellReuseIdentifier: "OptionCell")
```

Comment:
With this change, should we also remove the `OptionCell` identifier from storyboard? I only find one from `CreateSaveMap.storyboard` and not sure if it should be removed.

Reply:
No, It should not be removed from `CreateSaveMap.storyboard`. If it were removed, it would [break the sample](https://github.com/Esri/arcgis-runtime-samples-ios/blob/91459c9e25126e34eb52866985a83dca37a69759/arcgis-ios-sdk-samples/Maps/Create%20and%20save%20a%20map/CreateOptionsViewController.swift#L52). Reuse identifiers are not shared across storyboards.

Reply:
I see! Thanks for explanation. 👍